src/pages/api/user/getUserId.ts

```diff
@@ -0,0 +1,34 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { prisma } from '../../../lib/prisma';
+import { User } from '@prisma/client';
+
+const secret = process.env.NEXTAUTH_SECRET
+
+// API Inputs.
+export interface GetUserRequestType {
+    email: string;
+}
+
+// API Outputs.
+export interface GetUserResponseType {
+    user?: { id: string; }[];
+    error: string;
+}
+
+export default async function handler (
+    req: NextApiRequest,
+    res: NextApiResponse<GetUserResponseType>
+) {
+
+    // Find user information.
+    const user = await prisma.user.findMany({
+        select: {
+            id: true,
+        }
+    })
```

Comment:
Are you trying to get all ID's or is a query supposed to be here to get a specific one?

Reply:
He is querying all ids, originally meant for GetStaticPaths